pages/options/index.js

```diff
@@ -28,12 +28,18 @@ class OptionsPage extends Component {
         A: false,
         S: false,
       },
-      gender: '',
+      gender: 'All',
+      showHours: false,
+      sortByDistance: false,
+      showDropDown: false,
+      genders: ['All', 'Men', 'Women'],
```

Comment:
Genders is used only for display purposes. Is it better to just add each one of them in li?